lib/std/enumutils.nim

```diff
@@ -7,7 +7,9 @@
 #    distribution, for details about the copyright.
 #
 
-import macros
+import std/macros
+
+# xxx `genEnumCaseStmt` needs tests and runnableExamples
```

Comment:
Since it's `std / enumutils` (nothing less suffices, don't ever use Fusion... :-) ) you cannot leave out tests and examples here.

Reply:
but `genEnumCaseStmt` was introduced before this PR; it should be fixed in another followup PR (see future work on top that mentions this)